projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -2459,7 +2460,7 @@ public function __construct( $attributes, $content = null ) {
 		} elseif ( $post ) {
 			$attributes['id'] = $post->ID;
 			$post_author      = get_userdata( $post->post_author );
-			$default_to      .= $post_author->user_email;
+			$default_to      .= $post_author->user_email ? $post_author->user_email : '';
```

Comment:
Fixing 
```
PHP Warning: Attempt to read property "user_email" on bool in /wordpress/plugins/jetpack/11.8-a.13/modules/contact-form/grunion-contact-form.php on line 2462

```

Reply:
I don't think this will actually fix the warning. You'd need to do something like
```suggestion
			$default_to      .= isset( $post_author->user_email ) ? $post_author->user_email : '';
```
Fortunately `isset()` does the right thing for a case like that.